vm/src/stdlib/array.rs

```diff
@@ -545,6 +560,13 @@ mod array {
         fn byteswap(self) -> Self {
             Self(self.0.swap_bytes())
         }
+        fn to_object(self, vm: &VirtualMachine) -> PyObjectRef {
+            vm.ctx.new_str(
+                char::from_u32(self.0 as u32)
+                    .unwrap_or_default()
+                    .to_string(),
+            )
```

Comment:
is the convert safe?

Reply:
WideChar can contain invalid utf-8 character like surrogate. So I don't think so. To be safe, It must be decoded.

Reply:
Are we assuming the data in the array is decoded?

Reply:
I am not sure. I guess this is corresponding to WChar unicode object in CPython - which is deprecated same as this type array.
Even if it is regarded as a decoded string, because this is array, it still can contain surrogate character, which is not a valid utf8 string.
So my last comment was wrong. Regardless it is decoded or not, it can contains invalid utf8 character.